web/client/containers/HolyGrail.jsx

```diff
@@ -0,0 +1,65 @@
+const PropTypes = require('prop-types');
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+
+const { connect } = require('react-redux');
+
+const url = require('url');
+const urlQuery = url.parse(window.location.href, true).query;
+const PluginsUtils = require('../utils/PluginsUtils');
+const ConfigUtils = require('../utils/ConfigUtils');
+const BorderLayout = require('../components/layout/BorderLayout');
+
+const PluginsContainer = connect((state) => ({
+    mode: urlQuery.mode || (urlQuery.mobile || state.browser && state.browser.mobile ? 'mobile' : 'desktop'),
+    monitoredState: PluginsUtils.getMonitoredState(state, ConfigUtils.getConfigProp('monitorState'))
+}))(require('../components/plugins/PluginsContainer'));
+
+class HolyGrail extends React.Component {
```

Comment:
HolyGrail?

Reply:
I mean this https://en.wikipedia.org/wiki/Holy_grail_(web_design)